utils/parse.go

```diff
@@ -31,6 +31,14 @@ func ParseShell(input string) []string {
 
 	args := strings.FieldsFunc(input, f)
```

Comment:
since we are ignoring `'`, we probably don't need to "normalize" them in that first line of this function.